homeassistant/components/shelly/strings.json

```diff
@@ -170,6 +170,14 @@
       }
     }
   },
+  "exceptions": {
+    "device_communication_action_error": {
+      "message": "Device communication error occurred while calling the entity {entity} action for {device} device: {error}"
```

Comment:
We should not use method names in exception descriptions: https://github.com/home-assistant/core/pull/141005#discussion_r2006041017